lib/logos.spec.js

```diff
@@ -33,6 +34,15 @@ describe('Logo helpers', function() {
     )
   })
 
+  test(getSimpleIconColor, () => {
+    given({ icon: { hex: 'F7DF1E' }, style: undefined }).expect('F7DF1E') // light logo, dark background
+    given({ icon: { hex: 'F7DF1E' }, style: 'social' }).expect('#333') // light logo, light background
+    given({ icon: { hex: '004880' }, style: undefined }).expect('whitesmoke') // dark logo, dark background
+    given({ icon: { hex: '004880' }, style: 'social' }).expect('004880') // dark logo, light background
+    given({ icon: { hex: '00AFF0' }, style: undefined }).expect('00AFF0') // medium logo, dark background
+    given({ icon: { hex: '00AFF0' }, style: 'social' }).expect('00AFF0') // medium logo, light background
+  })
```

Comment:
It would be good to have a couple higher-level tests of this functionality. Perhaps you could call add a third test for `prepareNamedLogo()` with a result you've checked, or else a regular unit test that turns the base64 back into svg and checks that it includes `whitesmoke`.

This is "unconversion" code I wrote for #2473:

```js
const logoSvg = Buffer.from(
  getShieldsIcon({ name: 'github' }).replace('data:image/svg+xml;base64,', ''),
  'base64'
).toString('ascii')
```